rest-lib/src/main/scala/com/gu/mediaservice/lib/auth/provider/AuthorisationProvider.scala

```diff
@@ -3,6 +3,7 @@ package com.gu.mediaservice.lib.auth.provider
 import com.gu.mediaservice.lib.auth.Authentication.Principal
 import com.gu.mediaservice.lib.auth.SimplePermission
 import com.gu.mediaservice.lib.config.{CommonConfig, Provider}
+import com.gu.permissions.PermissionDefinition
```

Comment:
The Guardian permissions library has leaked into the abstraction here. This more or less defeats the point so I'm afraid I won't allow this. I suggest that as an alternative we create a temporary permission, perhaps `PinboardAssetsFeature` with an appropriate comment on it and an intention to get rid of it at some point in the future.

I plan to refactor the codebase in the coming months so that this sort of leak isn't possible (Guardian specific code lives in a separate sbt subproject so these common libraries can't see the dependency).

Reply:
my subconscious was telling me this might be problematic. happy to do that, as it reduces the complexity. Do you think the config model introduced in this PR should (for the time being) remove the notion of `app` so the `permission` field associated with a script in `scriptsToLoad` should just be a String (which will be checked against the `name` field of the `SimplePermission`?

Reply:
Or even hardcoded to just work with pinboard for now. I do think we should pursue a proper feature switch system to solve all of these problems, although it might be that when we do bring permissions with params back in that might also work for this.